client/src/components/HistoryViewer/HistoryViewer.js

```diff
@@ -54,6 +55,31 @@ class HistoryViewer extends Component {
     }
   }
 
+  /**
```

Comment:
```suggestion
  componentDidMount() {
    // Display a toast if there were any graphql errors
    const { graphQLErrors } = this.props;
    if (graphQLErrors.length > 0) {
      this.props.toastActions.error(i18n._t('Admin.UNKNOWN_ERROR', 'An unknown error has occurred.'));
    }
  }

  /**
```

Reply:
That doesn't work because on mount there are no errors - the errors come in the first update. This did point me in the right direction though, so thank you!